addons/mail/static/src/xml/web_kanban_activity.xml

```diff
@@ -6,7 +6,7 @@
         <!-- Dropdowns are created in JS to avoid some bugs, that's why the <a/> contains no args for the dropdown creation -->
         <a class="dropdown-toggle o-no-caret o_activity_btn" data-boundary="viewport" data-flip="false" data-bs-toggle="dropdown" role="button">
             <!-- span classes are generated dynamically (see _render) -->
-            <span t-att-title="widget.selection[widget.record.data.activity_state]" role="img" t-att-aria-label="widget.selection[widget.record.data.activity_state]"/>
+            <span t-att-title="widget.selection[widget.record.data.activity_state] ? widget.selection[widget.record.data.activity_state] : 'Schedule an activity'" role="img" t-att-aria-label="widget.selection[widget.record.data.activity_state]"/>
```

Comment:
Will that be translated ?

Reply:
Deprecated legacy code. No longer need to add a title for no activity.